app/controllers/account_controller.rb

```diff
@@ -85,7 +85,7 @@ def lost_password
       return
     else
       if request.post?
-        user = User.find_by_mail(params[:mail])
+        user = User.find_by(mail: params[:mail])
```

Comment:
Good catch sir. Looks like that method was finding them case-insensitive, so I will restore it.